.github/actions/deploy/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'Deploy'
+inputs:
+  name:
+    description: Name of helm release to deploy
+  namespace:
+    description: Target namespace
+  tag:
+    description: Docker tag to deploy
+    default: latest
+  chart_directory:
+    description: Location of helm chart
+  
+runs:
+  using: 'composite'
+  steps:
+
+  - name: helm deploy
+    uses: koslib/helm-eks-action@v1.24.1
+    with:
+      command: |
+        if [ "$ACTIONS_STEP_DEBUG" == "true" ]
+        then
+          set -x
+          jq --version || true
+        fi
+
+        # put env vars into values file
+        echo "environment: ${{ toJson(env) }}" > env-values.yaml
+        #env | grep '^HELM' | sed -e 's/^HELM_//' -e 's/=/: "/' -e 's/^/  /' -e 's/$/"/' >> env-values.yaml
```

Comment:
yes. This will pass all env vars of github worker into container. Idea is to prefix vars we want to pass into container with HELM_ so only required vars are passed. We couldn't do it before merge coz changing var names in repo secrets would broke pipelines on main branch